main/lsp/requests/hover.cc

```diff
@@ -6,19 +6,21 @@ using namespace std;
 
 namespace sorbet::realmain::lsp {
 
-string methodSignatureString(const core::GlobalState &gs, const core::TypePtr &retType,
-                             const core::DispatchResult &dispatchResult,
-                             const unique_ptr<core::TypeConstraint> &constraint) {
+string methodInfoString(const core::GlobalState &gs, const core::TypePtr &retType,
+                        const core::DispatchResult &dispatchResult,
+                        const unique_ptr<core::TypeConstraint> &constraint) {
     string contents = "";
     auto start = &dispatchResult;
     ;
     while (start != nullptr) {
         auto &dispatchComponent = start->main;
         if (dispatchComponent.method.exists()) {
             if (!contents.empty()) {
-                contents += " ";
+                contents += "\n";
             }
-            contents += methodDetail(gs, dispatchComponent.method, dispatchComponent.receiver, retType, constraint);
+            contents = absl::StrCat(
```

Comment:
Should we also display the `def` line when hovering over a method definition?

(I know it's a little silly since it's printed right there, but we already display the sig.)

Reply:
Will follow up with another PR with this change...